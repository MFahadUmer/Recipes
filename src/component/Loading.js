import React from 'react';
import ClockLoader from 'react-spinners/ClockLoader';

// eslint-disable-next-line react/prop-types
const Loading = ({ color, loading }) => (
  <div className="loading">
    <ClockLoader color={color} loading={loading} size={150} />
  </div>
);

export default Loading;
