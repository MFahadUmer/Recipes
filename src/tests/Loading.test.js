import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../component/Loading';

describe('Loading Component Tests', () => {
  it('matches the Loading component snapshot', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
