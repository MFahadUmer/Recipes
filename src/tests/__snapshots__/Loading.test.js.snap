// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Tests matches the Loading component snapshot 1`] = `
<div
  className="loading"
/>
`;
